.buildkite/release-pipelines/complete-code-freeze.yml

```
@@ -0,0 +1,32 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+---
+
+# Variables used in this pipeline are defined in `shared-pipeline-vars`, which is `source`'d before calling `buildkite-agent pipeline upload`
+
+agents:
+    queue: tumblr-metal
+
+steps:
+  - label: Complete Code Freeze
+    key: complete_code_freeze
+    plugins: [$CI_TOOLKIT_PLUGIN]
+    command: |
+      echo '--- :robot_face: Use bot for Git operations'
+      source use-bot-for-git wpmobilebot
+
+      echo '--- :git: Checkout release branch'
+      .buildkite/commands/checkout-release-branch.sh
```

2️⃣   Step 2 of my suggestion to avoid the need of the ugly workaround for `BUILDKITE_` prefix in the env var name:
```suggestion
      .buildkite/commands/checkout-release-branch.sh "$RELEASE_VERSION"
```

In this case the `$RELEASE_VERSION` env var will be resolved at `pipeline upload` time, which runs on the `upload` agent. Then when the actual job for "Complete Code Freeze" starts, the command it runs would thus already look like `.buildkite/commands/checkout-release-branch.sh 12.3`, so no more issue with `hostmgr` filtering out env vars when forwarding the command script to the VM.